app/index.js

```diff
@@ -1,18 +1,25 @@
 'use strict';
 
 // native modules
-var fs = require('fs');
-var http = require('http');
-var https = require('https');
-var EventEmitter = require('events').EventEmitter
+const fs = require('fs');
 
 // 3rd party modules
-var express = require('express');
-var logger = require('winston');
+const Express = require('express');
+const logger = require('winston');
+const SocketIO = require('socket.io');
+
+// application modules
+const express = require('./express')
+const Server = require('./server')
 const nconf = require('../config');
+const eventBus = require('./tools/eventBus');
+const models = require('./models');
+const routes = require('./routes');
+const AddonManager = require('./addons');
```

Comment:
I haven't use `.js`- seems that there is just couple `require()` lines which uses extension `.js`when importing application modules. I propose to use that way for now.